scripts/npm/apps-install/index.js

```diff
@@ -0,0 +1,39 @@
+/*
+This script handles installing node dependencies for a project that lives
+under cfgov-refresh, but has its own package.json. These projects appear
+under the ./cfgov/unprocessed/apps/ path.
+ */
+
+const fs = require( 'fs' );
+const exec = require( 'child_process' ).exec;
+
+const paths = require( '../../../config/environment' ).paths;
+
+// Aggregate application namespaces that appear in unprocessed/apps.
+// eslint-disable-next-line no-sync
+let apps = fs.readdirSync( `${ paths.unprocessed }/apps/` );
+
+// Filter out .DS_STORE directory.
```

Comment:
Trivial: This could be more broadly stated "Filter out hidden directories"

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3834